binrz/man/rizin.1

```diff
@@ -78,108 +82,110 @@ List supported IO plugins
 Map file at given address
 .It Fl M
 Disable demangling
-.It Fl n, nn
+.It Fl n , nn
 Control loading of RzBin info, with -nn only loading binary structures
 .It Fl N
-Disable user settings/projects from ~/.rizinrc, ~/.config/rizin/rizinrc and the scripts inside .../rizinrc.d/ directory
+Disables loading user settings from the rizinrc file
+(check the RZ_CONFIGHOME value in the rizin -H output).
 .It Fl NN
 Disable loading any scripts or plugins
 .It Fl q
 Quiet mode, suppresses prompt and quits after script execution
 .It Fl qq
-Quit before showing the prompt. Right after all the -e -c and -i arguments are evaluated
-.It Fl Q
-Same as q, but exiting without freeing RzCore, this produces leaks at exit time, but saves some precious ms to run the testsuite in fast mode
+Quit before showing the prompt.
+Right after all the -e -c and -i arguments are evaluated.
 .It Fl p Ar prj
 Set project file
 .It Fl r Ar rz-run
-Specify dbg.profile rz-run profile to use when spawning a program for debugging
+Specify dbg.profile rz-run profile to use when spawning for debugging
 .It Fl R Ar rule
-Specify custom rz-run directives without having to create a rz-run profile
+Specify custom rz-run directives without having to create a profile
 .It Fl s Ar addr
 Start seeking at this address
 .It Fl T
 Avoid computing the file hashes
 .It Fl u
 Set bin.filter=false to load rbin info without filtering names
 .It Fl v
-Show version information and exit (Use -qv to get just the version number)
+Show version information and exit.
+Use -qv to get just the version number.
 .It Fl V
 Show rizin library versions
 .It Fl w
 Open in write mode
 .It Fl x
 Open the file without the execute flag
 .It Fl X
-Same as -e bin.usextr=false, do not use extract plugins, useful for dyldcache
-.It Fl z, zz
-do not load strings or load them even in raw
+Same as -e bin.usextr=false, do not use extract plugins.
+.It Fl z , zz
+Do not load strings or load them even in raw
 .El
 .Sh ENVIRONMENT
-.Pp
-ANSICON:             ansicon's W & H of the buffer and w & h of the window in the form of: "WxH (wxh)"
-.Pp
-DEBUGINFOD_URLS:     e bin.dbginfo.debuginfod_urls - use alternative debuginfod server
-.Pp
-COLUMNS:             terminal columns to use
-.Pp
-RZ_ABORTLEVEL:       target log level/severity when to abort (0:DEBUG, 1:VERBOSE, 2:INFO, 3:WARN, 4:ERROR, 5:FATAL)
-.Pp
-RZ_CURL:             whether to use curl (for SSL support)
-.Pp
-RZ_DEBUG_ASSERT:     set a breakpoint when hitting an assert
-.Pp
-RZ_DEBUG_TOOL        debug tool to use when showing error messages and crash signal
-.Pp
-RZ_DYLDCACHE_FILTER: dyld cache filter (MacOS dynamic libraries location(s) at runtime)
```

Comment:
Here also the "at runtime" part should stay otherwise the brief description becomes incomplete?

Reply:
In which commit was the restore made?